Source/utils/display.cpp

```diff
@@ -235,6 +239,18 @@ bool SpawnWindow(const char *lpWindowName)
 #ifdef __vita__
 	scePowerSetArmClockFrequency(444);
 #endif
+#ifdef NXDK
+	{
+		Size windowSize = *sgOptions.Graphics.resolution;
+		VIDEO_MODE xmode;
+		void *p = nullptr;
+		while (XVideoListModes(&xmode, 0, 0, &p)) {
+			if (windowSize.width >= xmode.width && windowSize.height == xmode.height)
+				break;

```

Comment:
This logic could be better, but lets wait until XNDK gets the SDL calls implemented.